flashplayer.txt

```diff
@@ -0,0 +1,55 @@
+[Section]
+Name = Adobe Flash Player 32
+Version = 1.0.0
```

Comment:
why v1.0.0 if it is v32 actually? Why not upgrading any of the two existing Flash setups in Rapps, but creating a 3rd one?

Reply:
Now it's fixed. Updated to 32.0.0.101 (2018).

Reply:
Thx for fixing the version: Still the question: Why introducing a 3rd Flash player setup instead of updating the existing outdated ones?